Okta/OktaOidc/OktaOidc.swift

```diff
@@ -45,14 +50,17 @@ public class OktaOidc: NSObject {
                                callback: @escaping ((OktaOidcStateManager?, Error?) -> Void)) {
         currentUserSessionTask = task
 
-        task.signIn { [weak self] authState, error in
+        task.signIn(delegate: configuration.requestCustomizationDelegate) { [weak self] authState, error in
             defer { self?.currentUserSessionTask = nil }
             guard let authState = authState else {
                 callback(nil, error)
                 return
             }
             
             let authStateManager = OktaOidcStateManager(authState: authState)
+            if let delegate = self?.configuration.requestCustomizationDelegate {
+                authStateManager.restAPI = OktaOidcRestApi(delegate: delegate)
```

Comment:
Why just not:
```
authStateManager.restAPI. requestCustomizationDelegate = delegate
```
I would just avoid recreating `OktaOidcRestApi` outside of `OktaOidcStateManager` as you don't know all specifics how `OktaOidcRestApi` is created and configured inside `OktaOidcStateManager`

Reply:
I've removed recreating `OktaOidcRestApi` here, but there are few other places in this class where we creating `OktaOidcRestApi` object. Should we consider to change this as well?